src/modules/core/components/ListGroup/ListGroup.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { ReactNode } from 'react';
+
+import { getMainClasses } from '~utils/css';
+
+import styles from './ListGroup.css';
+
+interface Props {
+  children: ReactNode;
+}
+
+const displayName = 'ListGroup';
```

Comment:
Will the `ListGroup` eventually replace all other instances of lists? Ie: tasks, domains, permissions, etc...

As it seems to me these are basically a generalized list which can be filled with whatever content

Reply:
> Will the ListGroup eventually replace all other instances of lists? Ie: tasks, domains, permissions, etc...

I was imagining this to be the case - it's just much more simple to work with than a `table`, especially as we will (eventually) shift to support mobile. I think I'd like to develop it to be a little more robust first, though, and make this transition gradually.